app/employee-app/backend/service.bal

```diff
@@ -0,0 +1,1234 @@
+// Copyright (c) 2024, WSO2 LLC.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+import employee_app.authorization;
+import employee_app.database;
+import employee_app.email;
+import employee_app.types;
+
+import ballerina/cache;
+import ballerina/http;
+import ballerina/log;
+import ballerina/time;
+
+configurable email:EmailAlertConfig offerEmailConfig = ?;
+
+final cache:Cache userInfoCache = new ('capacity = types:CAPACITY, 'evictionFactor = types:EVICTION_FACTOR);
```

Comment:
```suggestion
final cache:Cache userInfoCache = new (capacity = types:CAPACITY, evictionFactor = types:EVICTION_FACTOR);
```

Reply:
This gives an error in both locations where uses `capacity` and `evictionFactor`,
`invalid value: For input string: "types:CAPACITY"`
`invalid value: For input string: "types:EVICTION_FACTOR"`

Reply:
Can you create a issue in the [balleirna-lang](https://github.com/ballerina-platform/ballerina-lang) repo. This shouldn't give any error.

Reply:
This is because in the constants we didn't specify the type.
`public const CAPACITY = 100;`
To maintain consistency, we've removed the types in the constants file. Is it a Ballerina issue that it's not identifying whether it's an int/float rather than a string?

Reply:
[Resolved](https://github.com/ballerina-platform/ballerina-lang/issues/43535)